docs/contributing-scalafmt.md

```diff
@@ -81,7 +81,7 @@ To build a native image of the command-line interface using
 
 - From the project root directory,
   - run `sbt cli/assembly`
-  - run `java -jar scalafmt-cli/target/scala-2.13/scalafmt.jar`, to execute recently built artifacts
+  - run `java -jar scalafmt-cli/jvm/target/scala-2.13/scalafmt.jar`, to execute recently built artifacts
```

Comment:
Unfortunately that's how cross building works in this case. We could probably override it for jvm?

Reply:
No suffix only works for for the sbt commands I think, so f.e. in the cross project instead of cliJVM/run we only need to run cli/run for the JVM platform in this branch. The idea was to keep the previously used commands intact.